docs/src/proposals/fee_transaction_priority.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Fee Transaction Priority
+---
+
+Additional fees were introduced to transactions as a method to allow users to bid for priority for
+their transactions in the leader's queue.
+
+Let the additional fee for a transaction `T` be defined as `F(T)`.
```

Comment:
`F(T) should be defined as  `(additional_fee + base_fee) / requested_cu` as fee-per-cu to prioritize transactions, instead of just using `additional_fee`, so 100 additional lamports for 1,000 CU transaction should have lower priority compare to 10 additional lamports for 10 CU transaction. The thing is the base_fee (eg signature, write lock etc) are bank dependent, as the fee_schedule changes over epochs.

Reply:
updated!